doc/reference/mobile.rst

```diff
@@ -0,0 +1,204 @@
+
+``showSnackBar(args)``
+......................
+
+Snackbars provide lightweight feedback about an operation. They show a brief message at the bottom of the screen on mobile and lower left on larger devices. Snackbars appear above all other elements on screen and only one can be displayed at a time.
+
+.. code-block:: javascript
+
+	mobile.methods.showSnackBar({'message': 'Message is deleted', 'btn_text': 'Undo'}).then(function(result){
+		if(result == true){
+			// Do undo operation
+		}else{
+			// Snack Bar dismissed
+		}
+	});
+
+.. image:: images/snackbar.png
+
+:**Parameters**:
+	
+	* ``message`` (string) - *required*
+		Message to show in snackbar
+
+	* ``btn_text`` (String) - *optional*
+		Action Button label in SnackBar
+
+:**Returns**:
+	
+	``True`` if User click on Action button
+
+	``False`` if SnackBar auto dismissed after some time
+
+``showNotification(args)``
+...........................
+
+A notification is a message you can display to the user outside of your application's normal UI. When you tell the system to issue a notification, it first appears as an icon in the notification area. To see the details of the notification, the user opens the notification drawer. Both the notification area and the notification drawer are system-controlled areas that the user can view at any time.
+
+.. code-block:: javascript
+	
+	mobile.showNotification({'title': 'Simple Notification', 'message': 'This is test of simple notification'})
+
+.. image:: images/mobile_notification.png
+
+:**Parameters**:
+	* ``title`` (string) - *required*
+		Set the title (first row) of the notification, in a standard notification.
+
+	* ``message`` (string) - *required*
+		Set the text (second row) of the notification, in a standard notification.
+
+``addContact(args)``
+....................
+
+Create device contact with given contact details.
+
+.. code-block:: javascript
+	
+	var contact = {
+		'name': 'Michel Fletcher',
+		'mobile': '9999999999',
+		'phone': '7954856587',
+		'fax': '765898745',
+		'email': 'michel.fletcher@agrolait.example.com',
+		'website': 'http://www.agrolait.com',
+		'street': '69 rue de Namur',
+		'street2': false,
+		'country_id': [21, 'Belgium'],
+		'state_id': false,
+		'city': 'Wavre',
+		'zip': '1300',
+		'parent_id': [8, 'Agrolait'],
+		'function': 'Analyst',
+		'image': '<<BASE 64 Image Data>>'
+	}
+
+	mobile.methods.addContact(contact);
+
+.. image:: images/mobile_contact_create.png
+
+:**Parameters**:
+	* ``contact dictionary`` (as shown in example) 
+		Valid contact details. Possible keys (name, mobile, phone, fax, email, website, street, street2, country_id, state_id, city, zip, parent_id, function and image)
+
+``scanBarcode()``
+.................
+
+The Barcode API detects barcodes in real-time, on device, in any orientation.
+
+It reads the following barcode formats:
+
+* 1D barcodes: EAN-13, EAN-8, UPC-A, UPC-E, Code-39, Code-93, Code-128, ITF, Codabar
+* 2D barcodes: QR Code, Data Matrix, PDF-417, AZTEC
```

Comment:
@xmo-odoo it is actually nice that this is specified, it doesn't hurt anyone to have it split.